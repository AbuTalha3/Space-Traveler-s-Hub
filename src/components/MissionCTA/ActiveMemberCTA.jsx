import React from 'react';
import './CTA.css';

const ActiveMemberCTA = () => (
  <div><button type="button" className="activeMemberCTA">Active Member</button></div>
);

export default ActiveMemberCTA;
