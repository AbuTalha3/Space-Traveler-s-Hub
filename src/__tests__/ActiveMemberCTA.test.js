import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import ActiveMemberCTA from '../components/MissionCTA/ActiveMemberCTA';

describe('tests Space Travel`s Hub App test', () => {
  it('renders the ActiveMember component', () => {
    const component = renderer.create(<ActiveMemberCTA />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
