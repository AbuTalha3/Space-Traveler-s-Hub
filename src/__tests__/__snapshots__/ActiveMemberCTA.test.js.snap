// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests Space Travel\`s Hub App test renders the ActiveMember component 1`] = `
<div>
  <button
    className="activeMemberCTA"
    type="button"
  >
    Active Member
  </button>
</div>
`;
